log4j-1.2-api/src/main/java/org/apache/log4j/builders/appender/ConsoleAppenderBuilder.java

```diff
@@ -107,6 +110,24 @@ public Appender parseAppender(final Element appenderElement, final XmlConfigurat
                             }
                             break;
                         }
+                        case FOLLOW_PARAM: {
+                            String value = getValueAttribute(currentElement);
+                            if (value == null) {
+                                LOGGER.warn("No value supplied for Follow parameter. Using default of {}", false);
+                            } else {
+                                follow.set(Boolean.valueOf(value));
+                            }
+                            break;
+                        }
+                        case IMMEDIATE_FLUSH_PARAM: {
+                            String value = getValueAttribute(currentElement);
+                            if (value == null) {
+                                LOGGER.warn("No value supplied for ImmediateFlush parameter. Using default of {}", true);
+                            } else if (!Boolean.getBoolean(name)) {
+                                LOGGER.warn("The value {} for ImmediateFlush parameter is not supported.", false);
+                            }
```

Comment:
FYI : This seems incomplete, where is the value used?

Reply:
Since the Log4j2 `ConsoleAppender` does not use the `immediateFlush` parameter (cf. [`ConsoleAppender.Builder#build`](https://github.com/apache/logging-log4j2/blob/0de646bde36d2c426988970314428d68b8858a5a/log4j-core/src/main/java/org/apache/logging/log4j/core/appender/ConsoleAppender.java#L218)), I only issue a warning that the value supplied by the user will be ignored.
Alternatively the Log4j2 `ConsoleAppender` could issue a warning if `immediateValue`, `bufferedIO` or `bufferSize` differ from the default values.